extensions/wikia/UserLogin/templates/UserSignupSpecial_signupForm.php

```diff
@@ -157,11 +156,6 @@
 				<p><?= wfMessage( 'usersignup-marketing-community' )->escaped() ?></p>
 			</div>
 			<div class="benefit">
-				<ul class="wikis">
```

Comment:
There's some css associated with this element that should be removed in UserSignup.scss.